posthog/admin.py

```diff
@@ -217,7 +217,8 @@ class TeamAdmin(admin.ModelAdmin):
                     "session_recording_sample_rate",
                     "session_recording_minimum_duration_milliseconds",
                     "session_recording_linked_flag",
-                    "updateCurrentTeam({ capture_console_log_opt_in: checked })" "data_attributes",
+                    "updateCurrentTeam({ capture_console_log_opt_in: checked })",
```

Comment:
bad copy pasta, sorry - 🤦 

I was following the trail of similar changes and bulk updating... should that `updateCurrentTeam(blah)` even be there?

```suggestion
```

Reply:
I still make Django up as I go (but don't tell anyone)

https://docs.djangoproject.com/en/4.2/ref/contrib/admin/#django.contrib.admin.ModelAdmin.fieldsets suggests that value is a list of fields so `updateCurrentTeam` feels like extra-double-bad copy-pasta on my part

Reply:
> I still make Django up as I go (but don't tell anyone)
> 
> https://docs.djangoproject.com/en/4.2/ref/contrib/admin/#django.contrib.admin.ModelAdmin.fieldsets suggests that value is a list of fields so `updateCurrentTeam` feels like extra-double-bad copy-pasta on my part

Haha, I haven't touched Django in forever. I think you're right, let me fix.